kitsune/wiki/tests/test_views.py

```diff
@@ -684,3 +685,144 @@ def test_it_works(self):
         res = self.client.get(url)
         eq_(res.status_code, 200)
         self.assertTemplateUsed(res, 'wiki/mobile/document.html')
+
+
+class FallbackSystem(TestCase):
+    "Check Article Wise Fallback Locale is working correclty"
```

Comment:
Doc strings should use triple quotes. Plus the wording here is a bit awkward. Something like this would be better:

```python
"""Check per-article fallback locales are working correctly."""
```